implementations/rust/ockam/ockam_command/src/trust_context/list.rs

```diff
@@ -0,0 +1,45 @@
+use clap::Args;
```

Comment:
All the `trust-context` commands were removed at https://github.com/build-trust/ockam/pull/7032